tests/vmi_multus_test.go

```diff
@@ -889,6 +878,35 @@ var _ = Describe("[Serial]SRIOV", func() {
 				return libnet.PingFromVMConsole(vmi2, cidrToIP(cidrA))
 			}, 15*time.Second, time.Second).Should(Succeed())
 		})
+
+		Context("With VLAN", func() {
+			const (
+				cidrVlaned1          = "192.168.0.1/24"
+				sriovVlanNetworkName = "sriov-vlan"
+			)
+
+			BeforeEach(func() {
+				createNetworkAttachementDefinition(sriovVlanNetworkName, tests.NamespaceTestDefault, sriovConfVlanCRD)
+			})
+
+			It("should be able to ping between two VMIs with the same VLAN over SRIOV network", func() {
+				_, vlanedVMI2 := createSriovVMs(sriovVlanNetworkName, sriovVlanNetworkName, cidrVlaned1, "192.168.0.2/24")
+
+				By("pinging from vlanedVMI2 and the anonymous vmi over vlan")
+				Eventually(func() error {
+					return libnet.PingFromVMConsole(vlanedVMI2, cidrToIP(cidrVlaned1))
+				}, 15*time.Second, time.Second).ShouldNot(HaveOccurred())
+			})
+
+			It("should NOT be able to ping between Vlaned VMI and a non Vlaned VMI", func() {
+				_, nonVlanedVMI := createSriovVMs(sriovVlanNetworkName, sriovnet3, cidrVlaned1, "192.168.0.3/24")
+
+				By("pinging between nonVlanedVMIand the anonymous vmi")
+				Eventually(func() error {
+					return libnet.PingFromVMConsole(nonVlanedVMI, cidrToIP(cidrVlaned1))
```

Comment:
Following this discussion https://github.com/kubevirt/kubevirt/pull/3479#discussion_r542764614
`PingFromVMConsole` is varidic function it enable you to pass arguments to ping command,
so you could pass -c 4 -w 7 to `PingFromVMConsole` so it will send more then one packet
https://github.com/kubevirt/kubevirt/blob/master/tests/libnet/ping.go#L37

Reply:
Thanks.
Done.

Reply:
This is not consistent with the existing tests and I see no particular reason to apply this enhanced ping for these new tests. You need to reason this.

Reply:
In that case, I think it worth a different PR, and we can consider to increase it for all pings.
@ormergi  what do you think?

Reply:
Ping test should include sending more then one packet to sense that the connection is stable
this is not stability check it just to see that the connection is not doping packets right after VM's are up.
Actually connectivity tests should include sending more then one packet.
About other tests we can align them in following PR.

Reply:
Sorry I missed your comment, I think we can start from VLAN tests and continue with others in another PR.

Reply:
> Ping test should include sending more then one packet to sense that the connection is stable
> this is not stability check it just to see that the connection is not doping packets right after VM's are up.
> Actually connectivity tests should include sending more then one packet.

Your reasoning here is that a packet may pass and another fail. How sending 5 or even more assure it will not occur after this?  If the link drops or is unstable we cannot detect here. It is also not the purpose of the test.
I would consider adding something like this only if it is seen in reality as a problem or we can recreate it.

Adding such a new logic is true for all, and we have no agreement on it. So this have the chance of leaving the tests in an inconsistent state.

Reply:
But I want to clarify something: Reasoning for this is not strong in this change or in general.
However, if we do change the defaults and announce that it is safer to perform the check using multiple packets, then I would not object as for me it does not matter much.

Reply:
So, to conclude, I am dropping this chamge drom this PR, and suggested a new one to increase the number of packets to all pings.
https://github.com/kubevirt/kubevirt/pull/4709